androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/Match.java

```diff
@@ -37,10 +34,9 @@
     public Map<String, String> redSwitch;
     public Map<String, String> blueSwitch;
     public Map<String, String> scale;
```

Comment:
How come these were not removed as well? Everything game specific should've been removed if I recall correctly